<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
<title>Index. microbiome 0.3.42</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="author" content="Leo Lahti and Jarkko Salojarvi">

<link href="css/bootstrap.css" rel="stylesheet">
<link href="css/bootstrap-responsive.css" rel="stylesheet">
<link href="css/highlight.css" rel="stylesheet">
<link href="css/staticdocs.css" rel="stylesheet">

<!--[if lt IE 9]>
  <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
<![endif]-->
  </head>

  <body>
    <div class="navbar">
  <div class="navbar-inner">
    <div class="container">
      <a class="brand" href="#">microbiome 0.3.42</a>
      <div class="nav">
        <ul class="nav">
          <li><a href="index.html"><i class="icon-home icon-white"></i> Index</a></li>
        </ul>
      </div>
    </div>
  </div>
</div>

    <div class="container">
      <header>
        
      </header>
      
      <div class="row">
  <div class="span8">
    <h1>microbiome</h1>

<p>microbiome R package - experimental beta release (<a href="http://microbiome.github.com">home
page</a>)</p>

<p>Check the <a href="https://github.com/microbiome/microbiome/wiki">wiki</a> for
installation instructions &amp; usage examples.</p>


    <h2>Help topics</h2>

    <h3></h3>
    
    
    <ul class="index">
        
      <li>
        <code><a href="add.heatmap.html">add.heatmap</a></code><br />add.heatmap
</li>
            
      <li>
        <code><a href="bootstrap.microbecount.html">bootstrap.microbecount</a></code><br />bootstrap.microbecount</li>
            
      <li>
        <code><a href="bootstrap.microbes.html">bootstrap.microbes</a></code><br />bootstrap.microbes</li>
            
      <li>
        <code><a href="calculate.hclust.html">calculate.hclust</a></code><br />calculate.hclust</li>
            
      <li>
        <code><a href="calculate.rpa.html">calculate.rpa</a></code><br />Description: RPA for HITChip</li>
            
      <li>
        <code><a href="calculate.stability.html">calculate.stability</a></code><br />Description: Stability analysis. Calculates average Pearson '</li>
            
      <li>
        <code><a href="check.dependencies.html">check.dependencies</a></code><br />Description: Load/install necessary packages and check OS</li>
            
      <li>
        <code><a href="check.wilcoxon.html">check.wilcoxon</a></code><br />Description: Calculate Wilcoxon test (unpaired; BH correction) for the specified sample groups.
</li>
            
      <li>
        <code><a href="chooseDir.html">chooseDir</a></code><br />Choosing (and creating) a directory</li>
            
      <li>
        <code><a href="choose.projects.html">choose.projects</a></code><br />Description: Select projects to analyze</li>
            
      <li>
        <code><a href="choose.samples.html">choose.samples</a></code><br />choose.samples</li>
            
      <li>
        <code><a href="Core2D.html">Core2D</a></code><br />Core2D</li>
            
      <li>
        <code><a href="Core3D.html">Core3D</a></code><br />Core3D</li>
            
      <li>
        <code><a href="core.sum.html">core.sum</a></code><br />Description: core.sum</li>
            
      <li>
        <code><a href="core.which.html">core.which</a></code><br />Description: core.which</li>
            
      <li>
        <code><a href="count.html">count</a></code><br />Description: count</li>
            
      <li>
        <code><a href="createCore.html">createCore</a></code><br />createCore</li>
            
      <li>
        <code><a href="cross.correlate.html">cross.correlate</a></code><br />Description: Cross-correlate input variables</li>
            
      <li>
        <code><a href="deconvolution.nonneg.html">deconvolution.nonneg</a></code><br />Description: Deconvolution</li>
            
      <li>
        <code><a href="detect.chip.html">detect.chip</a></code><br />Description: Detect the chip type (H/M/PITChip) from database name</li>
            
      <li>
        <code><a href="distance.matrix.html">distance.matrix</a></code><br />Description: Calculate distance matrix between the _columns_ of the
</li>
            
      <li>
        <code><a href="diversity.boxplot.html">diversity.boxplot</a></code><br />diversity.boxplot</li>
            
      <li>
        <code><a href="esort.html">esort</a></code><br />Description: Sort data frame dd by columns like: esort(dd, -z, b)</li>
            
      <li>
        <code><a href="estimate.diversity.html">estimate.diversity</a></code><br />estimate.diversity</li>
            
      <li>
        <code><a href="estimate.min.threshold.html">estimate.min.threshold</a></code><br />Description: determine threshold for bg correction</li>
            
      <li>
        <code><a href="estimateSpeciesPresence.html">estimateSpeciesPresence</a></code><br />Description: Returns the difference between the mean log10 signals of specific
</li>
            
      <li>
        <code><a href="expandCondition.html">expandCondition</a></code><br />Expands a list of "field", "value" pair lists into an SQL condition</li>
            
      <li>
        <code><a href="expandElement.html">expandElement</a></code><br />Expands one element (one "field", "value" pair list) from a list of "field", "value" pair lists</li>
            
      <li>
        <code><a href="FetchData.html">FetchData</a></code><br />Description: Fetch data from the database</li>
            
      <li>
        <code><a href="FetchHITChipAtlas.html">FetchHITChipAtlas</a></code><br />FetchHITChipAtlas</li>
            
      <li>
        <code><a href="fetch.measurements.html">fetch.measurements</a></code><br />Fetch measurements associated with one or more extractions from a phyloarray database</li>
            
      <li>
        <code><a href="fetch.projects.html">fetch.projects</a></code><br />fetch.projects</li>
            
      <li>
        <code><a href="fetch.sample.info.html">fetch.sample.info</a></code><br />fetch.sample.info</li>
            
      <li>
        <code><a href="fetch.samples.html">fetch.samples</a></code><br />Fetch samples from a phyloarray database</li>
            
      <li>
        <code><a href="GenerateSimulatedData.html">GenerateSimulatedData</a></code><br />Description: Generate toydata for the package</li>
            
      <li>
        <code><a href="get.bkg.params.html">get.bkg.params</a></code><br />Description: get background parameters</li>
            
      <li>
        <code><a href="get.doligo2.html">get.doligo2</a></code><br />Description: Calculate d.oligo2</li>
            
      <li>
        <code><a href="GetHclustParameters.html">GetHclustParameters</a></code><br />GetHclustParameters</li>
            
      <li>
        <code><a href="get.phylogeny.info.html">get.phylogeny.info</a></code><br />get.phylogeny.info</li>
            
      <li>
        <code><a href="get.probedata.html">get.probedata</a></code><br />Description: Get probedata</li>
            
      <li>
        <code><a href="get.probeset.html">get.probeset</a></code><br />Description: get probeset data matrix</li>
            
      <li>
        <code><a href="getPvalsFromLmMatrix.html">getPvalsFromLmMatrix</a></code><br />Description: Get Pvals From Lm Matrix</li>
            
      <li>
        <code><a href="get.sampleid.html">get.sampleid</a></code><br />Description: Pick sampleIDs from d.oligo column names</li>
            
      <li>
        <code><a href="hitchip.sig.lme.html">hitchip.sig.lme</a></code><br />Description: TBA</li>
            
      <li>
        <code><a href="importProjectFromPhyloarrayToR.html">importProjectFromPhyloarrayToR</a></code><br />Description: Import Project From Phyloarray To R. This function imports a
</li>
            
      <li>
        <code><a href="impute.html">impute</a></code><br />Impute missing values from a Gaussian.</li>
            
      <li>
        <code><a href="include.fig.html">include.fig</a></code><br />Description: brew embedded documentation helper function</li>
            
      <li>
        <code><a href="include.figs.html">include.figs</a></code><br />Description: brew embedded documentation helper function to include
</li>
            
      <li>
        <code><a href="level1TOlevel0.html">level1TOlevel0</a></code><br />Description: L1-L0 mappings</li>
            
      <li>
        <code><a href="level2TOlevel0.html">level2TOlevel0</a></code><br />Description: L2-L0 mappings</li>
            
      <li>
        <code><a href="level2TOlevel1.html">level2TOlevel1</a></code><br />Description: level2-level1 mappings</li>
            
      <li>
        <code><a href="levelmap.html">levelmap</a></code><br />levelmap</li>
            
      <li>
        <code><a href="list.clustering.metrics.html">list.clustering.metrics</a></code><br />Description: List clustering metrics</li>
            
      <li>
        <code><a href="list.color.scales.html">list.color.scales</a></code><br />Description: List color scales</li>
            
      <li>
        <code><a href="list.entities.html">list.entities</a></code><br />Description: list corresponding entities</li>
            
      <li>
        <code><a href="list.mysql.projects.html">list.mysql.projects</a></code><br />list.mysql.projects</li>
            
      <li>
        <code><a href="lmeMatrix.html">lmeMatrix</a></code><br />Description: Computes the given lme model for each variable (row) in the given data frame
</li>
            
      <li>
        <code><a href="lower.triangle.html">lower.triangle</a></code><br />Description:
</li>
            
      <li>
        <code><a href="make.abundancy.table.html">make.abundancy.table</a></code><br />make.abundancy.table</li>
            
      <li>
        <code><a href="matrix.qvalue.html">matrix.qvalue</a></code><br />matrix.qvalue</li>
            
      <li>
        <code><a href="microbiome-package.html">microbiome-package</a></code>(microbiome)<br />R package for microbiome studies</li>
            
      <li>
        <code><a href="mixingMatrix.html">mixingMatrix</a></code><br />Description: mixingMatrix</li>
            
      <li>
        <code><a href="mysql.format.html">mysql.format</a></code><br />Description: Format string vector to mysql query format</li>
            
      <li>
        <code><a href="n.phylotypes.per.oligo.html">n.phylotypes.per.oligo</a></code><br />Description: Check number of matching phylotypes for each probe</li>
            
      <li>
        <code><a href="oligo.bg.correction.html">oligo.bg.correction</a></code><br />Description: Calculate species summaries and possibly update d.oligo2</li>
            
      <li>
        <code><a href="outlierPvalue.html">outlierPvalue</a></code><br />Statistical test for outliers in a vector</li>
            
      <li>
        <code><a href="panel.stability.html">panel.stability</a></code><br />panel.stability</li>
            
      <li>
        <code><a href="PCA.dialogue.html">PCA.dialogue</a></code><br />Description: PCA dialogue window</li>
            
      <li>
        <code><a href="phyloarrayConnection.html">phyloarrayConnection</a></code><br />Tests whether the database connection is a phyloarray connection.
</li>
            
      <li>
        <code><a href="Phylogeneticenrichments.html">Phylogeneticenrichments</a></code><br />Phylogeneticenrichments</li>
            
      <li>
        <code><a href="phylogeny.html">phylogeny</a></code><br />phylogeny</li>
            
      <li>
        <code><a href="PhyloPlot.html">PhyloPlot</a></code><br />Description:
</li>
            
      <li>
        <code><a href="phylotype.rm.list.html">phylotype.rm.list</a></code><br />Description: Default list of removed phylotypes and oligos</li>
            
      <li>
        <code><a href="pick.training.samples.html">pick.training.samples</a></code><br />pick.training.samples</li>
            
      <li>
        <code><a href="plot_cumulative.html">plot_cumulative</a></code><br />plot_cunulative</li>
            
      <li>
        <code><a href="plotGroupPCA.html">plotGroupPCA</a></code><br />Description:</li>
            
      <li>
        <code><a href="plot.htrees.html">plot.htrees</a></code><br />plot.htrees
</li>
            
      <li>
        <code><a href="PlotMatrix.html">PlotMatrix</a></code><br />Visualize a matrix with one or two-way color scale.
</li>
            
      <li>
        <code><a href="plotMeanProfile.html">plotMeanProfile</a></code><br />Description: Plot Mean Profile</li>
            
      <li>
        <code><a href="plotOligoP.html">plotOligoP</a></code><br />Description: Plot oligoprofile figures without data retrieval from the db</li>
            
      <li>
        <code><a href="plotPCA.html">plotPCA</a></code><br />Description: Plot PCA with labels and explained variances</li>
            
      <li>
        <code><a href="PlotPhylochipHeatmap.html">PlotPhylochipHeatmap</a></code><br />PlotPhylochipHeatmap</li>
            
      <li>
        <code><a href="plotScatter.html">plotScatter</a></code><br />Description: plot scatter</li>
            
      <li>
        <code><a href="polish.phylogeny.info.html">polish.phylogeny.info</a></code><br />polish.phylogeny.info</li>
            
      <li>
        <code><a href="populate.checkboxes.html">populate.checkboxes</a></code><br />Description: low.level functions required by GUI</li>
            
      <li>
        <code><a href="populate.radiobuttons.html">populate.radiobuttons</a></code><br />Description: populate radiobuttons</li>
            
      <li>
        <code><a href="preprocess.chipdata.html">preprocess.chipdata</a></code><br />Description: Profiling preprocessing script</li>
            
      <li>
        <code><a href="preproc.Hitchip.html">preproc.Hitchip</a></code><br />Description: HITChip preprocessing function</li>
            
      <li>
        <code><a href="project.data.html">project.data</a></code><br />Description: Project high-dimensional data on two-dimensional plane by various methods</li>
            
      <li>
        <code><a href="prune16S.html">prune16S</a></code><br />Description: filter 16S data</li>
            
      <li>
        <code><a href="read.hitchip.html">read.hitchip</a></code><br />Description: read hitchip</li>
            
      <li>
        <code><a href="read.hitchip2.html">read.hitchip2</a></code><br />Description: Read HITChip file</li>
            
      <li>
        <code><a href="read.hitchip.prof008v3.4.html">read.hitchip.prof008v3.4</a></code><br />Description: read hitchip prof 008 v3.4</li>
            
      <li>
        <code><a href="read.in.data.html">read.in.data</a></code><br />Description: Read data</li>
            
      <li>
        <code><a href="ReadParameters.html">ReadParameters</a></code><br />Description: Define parameters in select box</li>
            
      <li>
        <code><a href="read.profiling.html">read.profiling</a></code><br />read.profiling</li>
            
      <li>
        <code><a href="read.profiling.010.html">read.profiling.010</a></code><br />read.profiling.010</li>
            
      <li>
        <code><a href="remove.background.html">remove.background</a></code><br />Description: simple background correction</li>
            
      <li>
        <code><a href="retrieve.probesets.html">retrieve.probesets</a></code><br />retrieve.probesets</li>
            
      <li>
        <code><a href="rsd.html">rsd</a></code><br />Relative standard deviation using paired measurements</li>
            
      <li>
        <code><a href="runPCA.html">runPCA</a></code><br />Description: PCA function</li>
            
      <li>
        <code><a href="run.profiling.script.html">run.profiling.script</a></code><br />run.profiling.script</li>
            
      <li>
        <code><a href="sampleOutliers.html">sampleOutliers</a></code><br />Outlier detection based on replicate measurements of a sample</li>
            
      <li>
        <code><a href="ScaleProfile.html">ScaleProfile</a></code><br />Description: Between-arrays normalization</li>
            
      <li>
        <code><a href="scaling.minmax.html">scaling.minmax</a></code><br />Description: Minmax scaling.</li>
            
      <li>
        <code><a href="simulate.hitchip.html">simulate.hitchip</a></code><br />Description: simulate.hitchip</li>
            
      <li>
        <code><a href="spatnorm.html">spatnorm</a></code><br />2D Spatial normalisation of microarrays</li>
            
      <li>
        <code><a href="species2levels.html">species2levels</a></code><br />Description: species-levels mappings</li>
            
      <li>
        <code><a href="Strip.html">Strip</a></code><br />Strip string i.e. remove spaces from the beginning and end</li>
            
      <li>
        <code><a href="submitDB.html">submitDB</a></code><br />Generic method for submission of calculated data to the database</li>
            
      <li>
        <code><a href="submitDB.sampleOutliers.html">submitDB.sampleOutliers</a></code><br />Delete previous outlier detection results for these samples. Also set
</li>
            
      <li>
        <code><a href="summarize.oligos.html">summarize.oligos</a></code><br />Description: summarize.oligos</li>
            
      <li>
        <code><a href="summarize.probesets.html">summarize.probesets</a></code><br />Description: Probeset summarization with various methods.</li>
            
      <li>
        <code><a href="summarize.probesets.species.html">summarize.probesets.species</a></code><br />Description: Probeset summarization with various methods.</li>
            
      <li>
        <code><a href="summarize.rawdata.html">summarize.rawdata</a></code><br />Description: Compress normalized raw data matrix into final probe-level matrix:
</li>
            
      <li>
        <code><a href="theme_bottom_border.html">theme_bottom_border</a></code><br />Description: ggplot2::ggplot2 theme</li>
            
      <li>
        <code><a href="threshold.data.html">threshold.data</a></code><br />Description: determine detection threshold for the data</li>
            
      <li>
        <code><a href="toydata-hitchip.html">toydata-hitchip</a></code><br />toydata-hitchip</li>
            
      <li>
        <code><a href="vwReg.html">vwReg</a></code><br />Description: Draw regression curve with smoothed error bars
</li>
            
      <li>
        <code><a href="WriteChipData.html">WriteChipData</a></code><br />Description: Write preprocessed data into the output directory</li>
            
      <li>
        <code><a href="WriteLog.html">WriteLog</a></code><br />Description: Write log file</li>
            
      <li>
        <code><a href="WriteMatrix.html">WriteMatrix</a></code><br />Description: Write matrix in tab file</li>
    
    </ul>
  </div>
  
  <div class="span3 offset1">
        
    <h2>Dependencies</h2>
    <ul>
      <li><strong>Depends</strong>: affy, brew, DBI, fields, ggplot2, gplots, limma, MASS, methods, mixOmics, nlme, NMF, outliers, plyr, preprocessCore, qvalue, reshape, R2HTML, RPA, svDialogs, tcltk, vegan</li>
      <li><strong>Imports</strong>: </li>
      <li><strong>Suggests</strong>: </li>
      <li><strong>Extends</strong>: </li>
    </ul>
     
    
  </div>
</div>
      
      <footer>
      <p class="pull-right"><a href="#">Back to top</a></p>
<p>Built by <a href="https://github.com/hadley/staticdocs">staticdocs</a>. Styled with <a href="http://twitter.github.com/bootstrap">bootstrap</a>.</p>
      </footer>
    </div>
  </body>
</html>